src/js/background.js

```diff
@@ -1,3 +1,3 @@
-chrome.runtime.onMessage.addListener(function(message, sender, sendResponse) {
-  if(message.closeThis) chrome.tabs.remove(sender.tab.id);
-});
\ No newline at end of file
+chrome.runtime.onMessage.addListener((message, sender) => {
+  if (message.closeFigmaTab) chrome.tabs.remove(sender.tab.id);
```

Comment:
Oh yeah I also renamed this just to be safe. `closeThis` felt kinda risky in case other messages are sent that match.